plugins/module_utils/network/eos/config/ospfv3/ospfv3.py

```diff
@@ -146,6 +146,12 @@ def _compare(self, want, have):
 
     def _global_compare(self, want, have):
         for name, entry in iteritems(want):
+            if name == "timers":
+                if entry.get("throttle"):
+                    self._module.fail_json(
```

Comment:
Surely you mean to warn, not fail? Otherwise you might as well just remove the option if it being set will fail the module